examples/graphbolt/rgcn/download.py

```diff
@@ -0,0 +1,608 @@
+    )
+    gb.numpy_save_aligned(f"{set_dir}/test_indices.npy", test_indices)
+    gb.numpy_save_aligned(f"{set_dir}/train_labels.npy", train_labels)
+    gb.numpy_save_aligned(f"{set_dir}/validation_labels.npy", validation_labels)
+    gb.numpy_save_aligned(f"{set_dir}/test_labels.npy", test_labels)
+
+
+def add_edges(edges, source, dest, dataset_size):
+    """This is for processing the edges in the graph and convert them to correct shape."""
+    for edge in edges:
+        print(f"\t Processing {edge} edge...")
+
+        old_edge_path = source + "/" + edge + "/" + "edge_index.npy"
+        new_edge_path = dest + "/" + edge + ".npy"
+        os.rename(src=old_edge_path, dst=new_edge_path)
+
+        # edge_array = np.memmap(new_edge_path, dtype='int32', mode='r',  shape=(num_edges[dataset_size][edge], 2))
+        edge_array = np.load(new_edge_path)
+        new_edge_array = edge_array.transpose()
+
+        assert new_edge_array.shape == (2, num_edges[dataset_size][edge])
+        assert np.array_equal(edge_array, new_edge_array.transpose())
+
+        gb.numpy_save_aligned(new_edge_path, new_edge_array)
+
+
+def process_feat(file_path, node_name, dataset_size):
+    """This is for processing the node features."""
+    # array = np.memmap(file_path, dtype='float32', mode='r',  shape=(num_nodes[dataset_size][node_name], 1024))
+    array = np.load(file_path)
+    assert array.shape == (num_nodes[dataset_size][node_name], 1024)
+    gb.numpy_save_aligned(file_path, array)
+
+    # Assert the shape and elements of the array are correct
+    # new_array = np.memmap(file_path, dtype='float32', mode='r',  shape=(num_nodes[dataset_size][node_name], 1024))
+    new_array = np.load(file_path)
+    assert new_array.shape == (num_nodes[dataset_size][node_name], 1024)
+    assert np.array_equal(array, new_array)
+
+
+def process_label(file_path, num_class, dataset_size):
+    """This is for processing the node labels."""
+    if (
+        num_class == 2983 and dataset_size == "full"
+    ):  # only this case label number changes
+        # array = np.memmap(file_path, dtype='int32', mode='r',  shape=(227130858, 1))
+        array = np.load(file_path)
+        assert array.shape[0] == 227130858
+    else:
+        # array = np.memmap(file_path, dtype='int32', mode='r',  shape=(num_nodes[dataset_size]["paper"], 1))
+        array = np.load(file_path)
+        assert array.shape[0] == num_nodes[dataset_size]["paper"]
+
+    gb.numpy_save_aligned(file_path, array)
+
+    # Assert the shape and elements of the array are correct
+    if num_class == 2983 and dataset_size == "full":
+        # new_array = np.memmap(file_path, dtype='int32', mode='r',  shape=(227130858, 1))
+        new_array = np.load(file_path)
+        assert new_array.shape[0] == 227130858
+        assert np.array_equal(array, new_array)
+    else:
+        # new_array = np.memmap(file_path, dtype='int32', mode='r',  shape=(num_nodes[dataset_size]["paper"], 1))
```

Comment:
```suggestion
        assert num_class == 19
        # new_array = np.memmap(file_path, dtype='int32', mode='r',  shape=(num_nodes[dataset_size]["paper"], 1))
```

Reply:
For this one, the else condition is for all cases other cases excluding the case (num_class=2983 && size == "full"), so I should comment out this assert?

Reply:
Don't all datasets work with both 2983 and 19?